build.mjs

```diff
@@ -166,6 +167,7 @@ const build = async () => {
   const purgeCSSResult = await new PurgeCSS().purge({
     content: ["dist/**/*.html"],
     css: ["dist/**/*.css"],
+    safelist: ["show"],
```

Comment:
I can see this change is on artichoke:trunk already, but why it shows the diff here? it's weird.

Reply:
I'm not sure what happened,  but I fixed up the branch in your fork.